backend/common.go

```diff
@@ -46,12 +46,30 @@ type DataSourceConfig struct {
 	Updated time.Time
 }
 
-// PluginConfig configuration for a plugin.
+// PluginConfig holds the configuration for a plugin instance.
+//
+// Grafana supports multiple organizations and only one plugin instance per Grafana organization. A plugin instance can have multiple data source instances.
+//
+// PluginConfig is attached to incoming requests to uniquely identify the Plugin instance the request belongs to.
+// If the request is a data source request, it also contains the configuration of the data source instance.
 type PluginConfig struct {
-	OrgID                   int64
-	PluginID                string
-	JSONData                json.RawMessage
+	// OrgID is the Grafana identifier of the Grafana organization this plugin instance belongs too.
+	OrgID int64
+
+	// PluginID is the unique identifer name of the plugin instance.
+	PluginID string
```

Comment:
Not positive on this one? Is id the global ID of the plugin, or the id of the plugin instance specific to the org?

Reply:
it is the id listed in `plugin.json`